tests/unit_tests/test_synthetic_dataset_image_operations_target.py

```diff
@@ -0,0 +1,24 @@
+import unittest
```

Comment:
Was this file intended to be left here? If so, please finish writing the test. Otherwise you can go ahead and delete the file